### ISSUE TITLE
Icons not showing  

### ISSUE THREAD

**Issue by ameerh** [cprodhomme/arctic_admin#39]
@cle61 I have integrated arctic admin with Rails5 API application. Its integrated fine and everything works well except the icons, they are not displaying. Please see attached screenshot.

I did import font-awesome. Here is my active_admin.css.scss file looks like:

@import "font-awesome";
@import "arctic_admin/base";

![screenshot 2018-07-10 14 50 43](https://user-images.githubusercontent.com/5346601/42502956-ce08d620-8450-11e8-9c75-4de9157ce40e.png)

**Comment by cle61** [comment]
You can remove the line : `@import "font-awesome";` it's useless.
which version of arctic_admin and font-awesome-rails do you use ?

**Comment by ameerh** [comment]
@cle61 Below are the gems in Gemfil:

gem 'arctic_admin'
gem "font-awesome-rails"
gem 'font-awesome-sass'


I have tried after removing @import "font-awesome"; but its still same. Please advise.

**Comment by cle61** [comment]
give me the exact version in the Gemfile.lock, please.

**Comment by ameerh** [comment]
Its  arctic_admin (1.5.0)

**Comment by cle61** [comment]
and "font-awesome-rails" & 'font-awesome-sass' ? I think you use font-awesome 5 and arctic_admin use font-awesome 4

**Comment by ameerh** [comment]
font-awesome-sass (5.0.13)
    font-awesome-rails (4.7.0.4)

**Comment by cle61** [comment]
arctic_admin use font-awesome 4 and you use font-awesome 5. I need to update arctic_admin but i don't have the time now.
If you don't use the gem `font-awesome-sass`, remove it

**Comment by ameerh** [comment]
@cle61 Yes I am not using it. I have removed it from gemfile. Its giving this error now:

Here is my active_admin.css.scss file looks like:

@import "font-awesome";
@import "arctic_admin/base";

![screenshot 2018-07-11 17 33 54](https://user-images.githubusercontent.com/5346601/42571463-ab6bd47e-8530-11e8-9a7f-9251ce64c08b.png)

**Comment by wasifhossain** [comment]
I think you also dont need `@import "font-awesome";` can you try again after removing the line? thanks

**Comment by ameerh** [comment]
@wasifhossain It shows the same error message even after removing `@import "font-awesome";`

**Comment by wasifhossain** [comment]
@ameerh so you are sure that you removed all font-awesome `import/require` variations as well as related gems still getting the error: `File to import not found or unreadable: font-awesome` ?

Logically if you remove all the references of a particular resource from js/css manifest files, it should not end up with errors like: <resource> not found. No?

**Comment by Divepit** [comment]
hello, I'm also struggling with this issue on production (not on development though) and I tried the suggested fixes and also have font-awesome-rails 4 installed. Is there a different solution for this?

**Comment by jcarlos7121** [comment]
Same issue here, already tried the suggested above

**Comment by cprodhomme** [comment]
I have created a new version `2.0.0` with the gem `font-awesome-sass`, do you have test ?

**Comment by EvanBrightside** [comment]
@cprodhomme the same problem / Arctic Admin 2.0.0 and no icons (font-awesome)

**Comment by cprodhomme** [comment]
Fix in the version `2.0.1` ! thanks for the repport ! :)

**Comment by mykeeouellet** [comment]
icons still not showing, is there someone that knows a fix in 2020 ?

**Comment by cprodhomme** [comment]
@mykeeouellet can you give me some context ? What is your gemfile ? Sass version

**Comment by cprodhomme** [comment]
Someone with the can test with the branch `fix/#57`

**Comment by EvanBrightside** [comment]
@cprodhomme im just tried branch fix/#57 / not working for me, no icons
its rails 6 / activeadmin 2.7.0 / arctic_admin 3.2.0

**Comment by EvanBrightside** [comment]
@cprodhomme for adding FA icons to arctic admin with (rails 6 / webpacker / activeadmin 2.7.0 / arctic_admin 3.2.0) we just need:
1. yarn add @fortawesome/fontawesome-free
2. Add `import "@fortawesome/fontawesome-free/js/all.js";` and `import "@fortawesome/fontawesome-free/css/all.css";` to /app/javascript/packs/active_admin.js

**Comment by fastengineer** [comment]
Any guide for rails 6 without yarn webpacker?
I am using sprockets.

**Comment by hgeorgilas** [comment]
@EvanBrightside Wouldn't work with `.scss` files, not sure if I did something wrong there.
I had to do the following to make the webfonts work

`app/javascript/stylesheets/active_admin.scss`
```
$fa-font-path: '~@fortawesome/fontawesome-free/webfonts';
@import '~@fortawesome/fontawesome-free/scss/fontawesome';
@import '~@fortawesome/fontawesome-free/scss/solid';
@import '~@fortawesome/fontawesome-free/scss/brands';
@import '~@fortawesome/fontawesome-free/scss/regular';
```

**Comment by CR1AT0RS** [comment]
I am using rails 6. This is what you need to do in order to fix missing icons:

- yarn add @fortawesome/fontawesome-free
- import "@fortawesome/fontawesome-free/css/all.css"; to /app/javascript/packs/active_admin.js

As @EvanBrightside pointed out thank you for that. But you don't need to add the full js library: import "@fortawesome/fontawesome-free/js/all.js"; it adds a 1.1 mb additional size you wanna import that via webpacked for only things  you need. This import will import the full JS and not recommended for production.

**Comment by mariagroves** [comment]
I have rails 6 / webpacker / activeadmin 2.7.0 / arctic_admin 3.2.0 and have tried the fix suggested by @EvanBrightside but still have a bug. 

![Screenshot 2020-10-02 at 11 03 40](https://user-images.githubusercontent.com/54529809/94906702-404c8580-049f-11eb-98f6-99a1a485a42c.png)

The fontawesome icons are loading but the dashboard has lost styling after:

1. `yarn add @fortawesome/fontawesome-free`
2. `import "@fortawesome/fontawesome-free/js/all.js"` and `import "@fortawesome/fontawesome-free/css/all.css"` in /app/javascript/packs/active_admin.js

I also tried the scss imports suggested by @hgeorgilas

**Comment by revskill10** [comment]
Doesn't work with sprockets
